src/pages/WorkspaceDetails/OverviewTab/index.tsx

```diff
@@ -132,6 +133,9 @@ export class OverviewTab extends React.Component<Props, State> {
               storageType={storageType}
               onSave={_storageType => this.handleStorageSave(_storageType)}
             />
+            <FormGroup label="Projects" fieldId="projects">
+              <div style={{ paddingTop: '5px', lineHeight: '30px' }}>{projects.join(', ')}</div>
```

Comment:
Doesn't it make sense to extract inline styles into CSS?

Reply:
It doesn't make sense. Because it is a temporary solution to have minimal information until the tab 'Projects' will be implemented.